src/std/main.ab

```diff
@@ -266,3 +272,15 @@ pub fun shell_var_get(name: Text): Text {
 pub fun shell_unset(name: Text): Null {
     $unset {name}$?
 }
+
+pub fun encode64(text: Text): Text {
+    return unsafe $echo -n "{text}" | base64$
+}
+
+pub fun decodeb64(encoded: Text): Text {
```

Comment:
what is the point of naming it `decodeb64`? does it stand for `decode` `b`ase `64`? in that case, its inconsistend naming with `encode64` which i guess stands for `encode` base `64`

Reply:
This was a human error. I'll put the b in base64 there. Do you think it would be okay? or give me another suggestion